### DISCUSSION TITLE
Has anyone tried WebGPU on an ARM device?

### DISCUSSION THREAD

**Discussion by hefeng-oppo** [gpuweb/gpuweb#2740] [Q&A] [answered]
I tried it with chromium browser, but there are many effect with dawn(chromium WebGPU Implemention)

https://bugs.chromium.org/p/dawn/issues/detail?id=1334

**Comment by kainino0x** [comment, marked as answer]
(readers: please note this repository is browser-independent, so browser-specific questions/discussions should be on the appropriate bug-trackers, like the issue filed/linked above.)

Chrome's implementation is known not to work just yet on Android.

Firefox's implementation [has been known to work on Android](https://github.com/gpuweb/gpuweb/wiki/Implementation-Status#firefox-and-servo). However, I don't know the current status. Firefox's implementation is currently a bit more outdated on all platforms, so to try it out you'll need to use an older, incompatible "snapshot" of the API.

**Reply by hefeng-oppo** [reply]
I'll try it with Firefox, thanks.
